### ISSUE TITLE
Error: Unsupported argument on bigip_ltm_policy

### ISSUE THREAD

**Issue by howsTricks** [terraform-providers/terraform-provider-bigip#176]
When attempting to use an example of bigip_litm_policy ( https://www.terraform.io/docs/providers/bigip/r/bigip_ltm_policy.html ), using the following tf code, I get the error message outlined below:

```
provider "bigip" {
    address = "x.x.x.x"
    username = "xxxx"
    password = "xxxx"
}
resource "bigip_ltm_policy" "test-policy" {
  name           = "my_policy"
  strategy       = "first-match"
  requires       = ["http"]
  published_copy = "Drafts/my_policy"
  controls       = ["forwarding"]
  rule {
    name = "rule6"

    action = {
      tm_name = "20"
      forward = true
      pool    = "/Common/mypool"
    }
  }
  depends_on = ["bigip_ltm_pool.mypool"]
}

resource "bigip_ltm_pool" "mypool" {
    name = "/Common/mypool"
    monitors = ["/Common/http"]
    allow_nat = "yes"
    allow_snat = "yes"
    load_balancing_mode = "round-robin"
}
```
```
PS C:\xxx> terraform plan

Error: Unsupported argument

  on testPolicy.tf line 15, in resource "bigip_ltm_policy" "test-policy":
  15:     action = {

An argument named "action" is not expected here. Did you mean to define a
block of type "action"?

```

Using:
Terraform v0.12.9
provider.bigip v0.12.4

**Comment by RavinderReddyF5** [comment]
@howsTricks 
Documentation listed in (https://www.terraform.io/docs/providers/bigip/r/bigip_ltm_policy.html ) is based on terraform v0.11.4, there is change in configuration for terraformv0.12.

Can you please run : **terraform 0.12upgrade** before applying terraform plan, this will convert your .tf config to terraform v0.12 compatible

**Comment by howsTricks** [comment]
Ah yes, that worked, thanks!

**Comment by papineni87** [comment]
This was fixed in the 1.1.0 release. If you are still having a problem with that release, please reopen this issue or file a new one.
